traffic_ops/app/lib/MojoPlugins/Daemonize.pm

```diff
@@ -41,12 +41,6 @@ sub register {
 				# This is the first child
 				$self->inactivity_timeout(0);
 				POSIX::setsid();
-				open( STDIN, "< /dev/null" )
-					|| confess("Can't read /dev/null: $!");
```

Comment:
Why are you removing these lines ?
They make sure that any accidental input/output activity in the child process will not interfere with the parent process.
It is a standard thing to do when spawning child processes.